### ISSUE TITLE
Tag for previous release (20181116).

### ISSUE THREAD

**Issue by Zimmi48** [charguer/tlc#4]
In nixpkgs, we have packaged the latest release, but also release `20181116`, that can be found at https://tlc.gforge.inria.fr/releases/tlc-20181116.tar.gz (to provide a version compatible with Coq 8.6-8.9). Unfortunately, there is no `20181116` tag, that would allow tracing back this release to a specific commit (although I guess it should be 4babc16c2b37555f3f314bb9617a62ddbd347929 since it's the only one on this date). We would like to retrieve all the releases from GitHub from now on, so we would appreciate if you could confirm that this is the commit to use by putting the `20181116` tag on it.

**Comment by charguer** [comment]
The branch coq-8.9 of TLC should compile with TLC 8.8.2 and 8.9, and I think compiles with warning on 8.10 and 8.11.

I think the one commit you mention is the right one, if you want to match the TLC package. Only François Pottier could confirm for sure.

**Comment by Zimmi48** [comment]
Thanks for your answer! @fpottier Would you mind confirming?

**Comment by fpottier** [comment]
Hello. The correct link for the old archive is http://tlc.gforge.inria.fr/releases/tlc-20181116.tar.gz (the one you give, which uses https, does not work). I think that the commit that you have identified is correct. I have just tagged it 20181116 in the github repository, so the link https://github.com/charguer/tlc/archive/20181116.tar.gz now works. This new archive is not identical to the old one, though; it contains more files, because back then we used to remove some unneeded files from the publicly released archives. I hope this helps!
